package.json

```diff
@@ -10,7 +10,7 @@
     "build": "quasar build",
     "lint": "eslint --ext .js,.ts,.vue ./",
     "test": "echo \"No test specified\" && exit 0",
-    "pre-deploy": "npm run build"
+    "prepare": "husky install"
```

Comment:
Husky git hook now gets installed when you do `npm install`, some people dont like this gets turned on. 

Can this be made manual and added to README.dev.md#linting?

Something like:

```markdown
To run linting on commit, you can install a git commit hook with

```shell
npx husky install
```

```